src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -38,6 +39,9 @@ def __init__(self, revision, cache_root, client_id, access_token):
 
         assert os.path.isdir(cache_root), 'Cache root {} is not a dir.'.format(cache_root)
         self.repo_dir = os.path.join(cache_root, 'mozilla-central')
+        self.temp_dir = tempfile.mkdtemp()
```

Comment:
Nit: this could just be a local variable, it's not used anywhere else than in this function